lib/adapters/http.js

```diff
@@ -279,15 +279,29 @@ module.exports = function httpAdapter(config) {
 
     // Handle request timeout
     if (config.timeout) {
+      // This is forcing a int timeout to avoid problems if the `req` interface doesn't handle other types.
+      var timeout = parseInt(config.timeout, 10);
```

Comment:
Can we please change this to `let timeout=`

Reply:
I tryed this, but lint doesn't allow this :/

Reply:
Forgot no es6 so that's fine leave it as is.